Source/analyzer/pattern.cpp

```diff
@@ -397,12 +403,12 @@ bool matchesSolidPattern()
 		}
 	}
 
-	std::cout << sgGameInitInfo.dwSeed << " possible game Seed for dlvl " << (int)currlevel << std::endl;
+	std::cout << "PATTERN: Potential dungeon level " << (int)currlevel << " seed: " << sgGameInitInfo.dwSeed << std::endl;
```

Comment:
The seed has to go first for the output to be valid as an input for `--seeds`.
```suggestion
	std::cout << sgGameInitInfo.dwSeed << " possible game Seed for dlvl " << (int)currlevel << std::endl;
```